src/main/java/racingcar/domain/Car.java

```diff
@@ -2,11 +2,20 @@
 
 import static racingcar.domain.Moving.*;
 
-public class Car {
+import java.util.Objects;
+
+public class Car implements Comparable<Car> {
+    private static final Distance ZERO = new Distance(0);
+    private static final int GREATER = 1;
+    private static final int EQUALS = 0;
+    private static final int LESS = -1;
+    
+    private final String name;
     private Distance total;
 
     public Car(String name) {
-        total = new Distance(name);
+        this.name = name;
+        total = ZERO;
```

Comment:
`private static final Distance ZERO = new Distance(0);`
ZERO를 Car 내부에서 생성해서 가지고 있기 보다 Distance 내부에 만들어 두고 사용하면 어떨까요?
이렇게 사용할 수 있을 것 같아요
```suggestion
        total = Distance.ZERO;
```

Reply:
인호님 말씀해주신 내용들도 다 맞다고 생각합니다 😃 
경우에 따라 오버엔지니어링으로 볼 수 있을 것 같아요! 
저는 Car 라는 객체의 SRP 관점에서 의견을 드렸습니다! SRP 원칙은 변경되는 이유가 하나여야 한다고 하는데요,
Distance의 생성규칙이 변경되면 Car에서도 같이 변경이 필요할 것 같아요. SRP를 지키기 위해 Distance의 상수는 Distance가 
직접 제공하고, Car는 Distance.ZERO 라는 상수만 변경되지 않는다면 ZERO가 어떻게 생성되는지 몰라도 될 것 같아요!